### PR TITLE
DOC-3716 Create alerts page for ongoing Log4j announcements and patches

### PR DESCRIPTION
**Pull request by benpesterr3**

### CODE REVIEW

File: content/en/release-notes/_index.md [hunk 1]
```diff
@@ -12,6 +12,53 @@ version: 'releases'
...
+
+{{< note >}}
+These patch releases are valid for the stated supported versions of Corda and CENM only. If you are not using a supported version of Corda or CENM, please upgrade to one of the above versions.
+{{< /note >}}
+
+## What you can do now
+
+While you wait for the release of the required emergency patch, you can apply one of the following steps to mitigate the threat implied by the log4j Zero-day vulnerability, as referenced in https://nvd.nist.gov/vuln/detail/CVE-2021-44228.
+
+### For Corda 4.3.8 and above, and CENM 1.3 and above
```
Comment (mnesbit) [inline content/en/release-notes/_index.md:46]: Would be more correct to say Corda 4.3 and above as the original GA release is eligible for the tactical fix

File: content/en/release-notes/_index.md [hunk 2]
```diff
@@ -12,6 +12,53 @@ version: 'releases'
...
 
+# Apache Log4j announcement
+
+Following news of [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
+
+Your individual CorDapps do not need to be patched as they inherit Apache Log4j from the Corda runtime.
+
+Check the list below for expected patch release dates for your version of Corda or CENM. Use the [mitigation guide](#what-you-can-do-now) to reduce your risk before upgrading to the new patch.
+
+### Apache Log4j patch release timetable
```
Comment (mnesbit) [inline content/en/release-notes/_index.md:23]: This is ambiguous and should say: Corda patch release timetable for CVE-2021-44228 issue

File: content/en/release-notes/_index.md [hunk 3]
```diff
@@ -12,6 +12,53 @@ version: 'releases'
...
+| CENM 1.2.4      | Dec 14      | No |
+| CENM 1.5.2      | Dec 15      | Yes |
+| CENM 1.3.3      | Dec 16      | Yes |
+| CENM 1.4.2      | Dec 17      | Yes |
+
+{{< note >}}
+These patch releases are valid for the stated supported versions of Corda and CENM only. If you are not using a supported version of Corda or CENM, please upgrade to one of the above versions.
+{{< /note >}}
+
+## What you can do now
```
Comment (mnesbit) [inline content/en/release-notes/_index.md:42]: This may want to be nearer the start of document

**Review by mnesbit — DISMISSED** [review]
LGTM

**Review by erinking-r3 — DISMISSED** [review]
LGTM

File: content/en/release-notes/_index.md [hunk 4]
```diff
@@ -12,6 +12,53 @@ version: 'releases'
 title: Releases
 ---
 
+# Apache Log4j announcement
+
+Following news of [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), new patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM are in progress.
```
Comment (erinking-r3) [inline content/en/release-notes/_index.md]: In response to the [Apache Log4j 2's vulnerability to attack](https://nvd.nist.gov/vuln/detail/CVE-2021-44228), R3 is releasing patches for all supported versions of Corda Open Source, Corda Enterprise, and CENM.

File: content/en/release-notes/_index.md [hunk 5]
```diff
@@ -12,6 +12,53 @@ version: 'releases'
...
+
+Your individual CorDapps do not need to be patched as they inherit Apache Log4j from the Corda runtime.
```
Comment (erinking-r3) [inline content/en/release-notes/_index.md:19]:
```suggestion
You do not need to patch CorDapps— they inherit Apache Log4j from the Corda runtime.
```

**Review by michelemarsden — DISMISSED** [review]
LGTM

**Review by michelemarsden — APPROVED** [review]
LGTM

**Pull request merged**